Platform/ApollolakeBoardPkg/BoardConfig.py

```diff
@@ -134,12 +136,12 @@ def __init__(self, *args, **kwargs):
 
 		self.PLD_HEAP_SIZE        = 0x08000000
 
-		self.FWUPDATE_SIZE        = 0x00030000
+		self.FWUPDATE_SIZE        = 0x00020000
 		self.CFGDATA_SIZE         = 0x00004000
 		self.CFG_DATABASE_SIZE    = self.CFGDATA_SIZE
 		self.MRCDATA_SIZE         = 0x00004000
 		self.VARIABLE_SIZE        = 0x00002000
-		self.SPI_IAS1_SIZE        = 0x00150000
+		self.SPI_IAS1_SIZE        = 0x00001000
```

Comment:
make sure you check it can still work for ACRN case after change the IAS layout.

Reply:
OK. will update this patch to make IAS1 size to smaller only when UEFI payload is enabled. so that it will not impact ACRN case.